### PR TITLE
New package: gutenprint

### PR DESCRIPTION
**Pull request by Anachron**
Top quality printer drivers for POSIX systems

### CODE REVIEW

**Comment by Anachron** [comment]
Ping @ebfe @the-maldridge @Vaelatern @lemmi

File: srcpkgs/gutenprint/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+checksum="a42489fd7d6f2e4320811754bb80b0cd845db6632b52f819fbc2b4b75057341b"
+configure_args="--disable-rpath \
+	--enable-samples --disable-static --disable-static-genppd --enable-cups-ppds \
+	--enable-simplified-cups-ppds=only --enable-translated-cups-ppds \
+	--enable-globalized-cups-ppds"
+nocross="https://sourceforge.net/p/gimp-print/mailman/message/34782748/"
+
+post_install() {
+	mv "${DESTDIR}"/usr/sbin/* "${DESTDIR}"/usr/bin/
+	rm -rf ${DESTDIR}/usr/sbin/
```
Comment (Vaelatern) [inline srcpkgs/gutenprint/template]: This probably should be a `rmdir`

**Comment by Vaelatern** [comment]
Commit message needs to be formatted correctly

**Comment by Vaelatern** [comment]
This gets close!

**Comment by Anachron** [comment]
Something went wrong, hang on ...

**Comment by Vaelatern** [comment]
You need to `git push -f`

**Comment by Anachron** [comment]
No idea why it would cause this conflict. Git is stupid.

**Pull request merged**